projects/plugins/crm/.gitattributes

```diff
@@ -19,13 +19,22 @@ node_modules         export-ignore
 # Files to include in the mirror repo, but excluded via gitignore
 # Remember to end all directories with `/**` to properly tag every file.
 # /src/js/example.min.js  production-include
-build/**                                        production-include
 jetpack_vendor/**                               production-include
 vendor/autoload.php                             production-include
 vendor/autoload_packages.php                    production-include
 vendor/automattic/**                            production-include
 vendor/composer/**                              production-include
 vendor/jetpack-autoloader/**                    production-include
+vendor/dompdf/**                                production-include
+vendor/phenx/**                                production-include
```

Comment:
I note that dompdf has two more deps (masterminds/html5 and sabberworm/php-css-parser) that aren't being included yet. They should probably be added too, unless we're sure that no CRM code path will ever use the dompdf features that need them.